filebeat/filebeat.yml

```diff
@@ -62,6 +62,35 @@ filebeat.inputs:
   # Note: After is the equivalent to previous and before is the equivalent to to next in Logstash
   #multiline.match: after
 
+# filestream is an experimental input. It is going to replace log input in the future.
+- type: filestream
```

Comment:
agent probably does not support this input do we need to include it?

Reply:
We should not include it at the moment. The input is experimental, so there might be some changes.